samples/crypto/rsa/README.rst

```diff
@@ -26,13 +26,13 @@ The sample performs the following operations:
 1. Initialization:
 
    a. The Platform Security Architecture (PSA) API is initialized.
-   #. A random RSA key pair is generated and imported into the PSA crypto keystore.
-   #. The public key of the RSA key pair is imported into the PSA crypto keystore.
+   #. A RSA key pair is imported into the PSA crypto keystore.
```

Comment:
It is unfortunate that we have to rely on a pre-generated key, but I think it should be mentioned that this key is not usable in any end-products. Maybe this text should be given in italics

```suggestion
   #. A RSA key pair is imported into the PSA crypto keystore.
   #. *Note that this is a test key and it should never be used in a product!*
```